diesel_cli/src/pretty_printing.rs

```diff
@@ -1,183 +0,0 @@
-use std::fmt::{Write, Error as FmtError};
```

Comment:
I guess this file wasn't removed in the PR that changed how pretty printing was done ?